admin/dataAccess.php

```diff
@@ -269,7 +269,6 @@ function edit_unit(&$id, &$unitName, &$unitDescription) {
     function insert_unit_offering(&$unitID, &$unitSemester, &$unitYear, &$unitConvenor) {
         $conn = oci_connect('web_app', 'password', 'dbi-tcs.c0nvd8yryddn.us-west-2.rds.amazonaws.com/DBITCS');
```

Comment:
Throw that AWS URL into an environment variable, we don't want staging links exposed to the public.

Reply:
Lmao like we had a staging environment

Reply:
Test on prod

Reply:
ok here's another question then, why were we hosting in the `us-west-2` region? shouldn't we have been using `ap-southeast-2`?

Reply:
Ahh you see, this was just great foresight from myself. As you might but probably don't remember, AWS Sydney was down on the day of our presentations, but US West continued to function. So on the morning of our presentations, our site worked and nobody elses did.

Also cause I didn't find that setting until after it was set up lol